ui-components/src/global/homepage.scss

```diff
@@ -13,7 +13,7 @@
   }
 
   h2 {
-    @apply text-base;
+    @apply text-xl;
```

Comment:
Please make this change in the style overrides for `sites/public` so that our `ui-components` are still in sync with upstream. Also note where else `h2` is used so that we can check to see if we should only increase the size on the front page or for all places where we're using `h2`.

Reply:
Reworked for only on public site and for this one instance.  That way (confirmed other pages) all other H2 are not impacted.